app/src/main/res/layout/info_window_layout.xml

```diff
@@ -1,4 +1,91 @@
+    android:padding="10dp"
+    android:background="@android:drawable/dialog_holo_light_frame">
+
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:orientation="horizontal">
+        <TextView
+            android:id="@+id/info_window_person"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Name"
+            android:textSize="20sp"
+            android:textAppearance="?android:attr/textAppearanceSmall"
+            android:textColor="@android:color/black" />
+        <TextView
+            android:id="@+id/info_window_socialSituation"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="Alone"
+            android:textAppearance="?android:attr/textAppearanceButton"
+            android:textColor="@android:color/black"
+            android:layout_marginStart="10sp"
+            android:textSize="10sp"
+            android:textStyle="bold" />
+
+
+    </LinearLayout>
+
+    <TextView
+        android:id="@+id/info_window_date"
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_below="@+id/userName"
+        android:textStyle="italic"
+        android:layout_marginTop="25dp"
+        android:text="Fri Mar 24 22:26:55 MDT 2017"
+        android:textColor="@color/gray" />
+
+
+    <LinearLayout
+        android:layout_width="wrap_content"
+        android:layout_height="wrap_content"
+        android:layout_centerInParent="true">
+
+        <TextView
+            android:id="@+id/info_window_description"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="My dog and I had a good walk today"
+            android:textColor="@android:color/black" />
+
+        <TextView
+            android:id="@+id/info_window_append"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:text="-Feeling Happy"
+            android:textAppearance="?android:attr/textAppearanceButton"
+            android:textColor="@android:color/black"
+            android:textStyle="bold|italic" />
+        <ImageView
+            android:id="@+id/info_window_emojiImage"
+            android:adjustViewBounds="true"
+            android:layout_width="wrap_content"
+            android:layout_height="wrap_content"
+            android:maxWidth="30dp"
+            android:maxHeight="30dp"
+            android:layout_marginStart="15sp"
+            android:scaleType="fitCenter"
+            />
+
+    </LinearLayout>
+    <ImageView
+        android:id="@+id/info_window_image"
+        android:layout_width="200sp"
+        android:layout_height="200sp"
+        android:adjustViewBounds="true"
+        android:layout_gravity="center"
+        />
+</LinearLayout>
+
```

Comment:
this XML is not the latest version yet, need to be changed after merge with @forgeno